src/code42cli/cmds/search/extraction.py

```diff
@@ -124,3 +116,54 @@ def create_time_range_filter(filter_cls, begin_date=None, end_date=None):
 
     elif end_date and not begin_date:
         return filter_cls.on_or_before(end_date)
+
+
+def create_send_to_handlers(
```

Comment:
This could be more dry..

Some shared code between both methods that create handlers

Reply:
I created `_extract_events` and `output_processor` to use shared code but then that is not acceptable [here](https://github.com/code42/code42cli/pull/137#discussion_r475769356) and I was advised to separate them out.

Anyways I will be doing the suggested changes.

Reply:
Done some changes.

Reply:
I think it is better here-- less args needed

Reply:
Thanks

Reply:
When I made that original comment, that shared method was not actually being shared. Creating the separate handlers makes it so that we don't need to pass in an `output_processor`. I found that to be confusing because `handle_response` is an output processor, and if you need a different output processor, you should create extra handlers, which you did (thank you). That does not mean the methods that create handlers can't share code or that the methods like `send_to` and `search` can use shared functions either.  I am sorry for that confusion, not trying to give you contradicting advice..